src/crypto/yespower-bitzeny.c

```diff
@@ -0,0 +1,50 @@
+/*-
+ * Copyright 2009 Colin Percival
+ * Copyright 2013-2018 Alexander Peslyak
```

Comment:
Please don't forget to drop these copyright statements from yespower-bitzeny.c.

Reply:
removed :)